platforms/iOS/vm/OSX/sqSqueakOSXApplication.m

```diff
@@ -166,7 +168,7 @@ - (int) parseArgument: (NSString *) argData peek: (char *) peek
 	/* Options with no arguments */
 
 	NS_DURING;
-		if ([argData compare:  VMOPTIONOBJ("psn_") options: NSLiteralSearch range: NSMakeRange(0,5)] == NSOrderedSame) {
+		if ([argData compare:  VMOPTIONOBJ("psn_") options: NSLiteralSearch range: NSMakeRange(0,VMOPTIONLEN(5))] == NSOrderedSame) {
```

Comment:
Side comment: using `VMOPTIONOBJ` makes no sense for "psn_". It will always have exactly one dash, so using VMOPTIONOBJ (and the otherwise useful VMOPTIONLEN) makes no sense

Reply:
Good point. OSX doesn't care about the number of dashes we have. I don't like the code. We should strip the dashes before parsing the arguments but that would have been a bigger change and outside the timebox I had. :(

Reply:
No problem, just a side comment to not forget